app.config.ts

```diff
@@ -0,0 +1,85 @@
+const projectName = 'hypecycle';
+const appName =
+  process.env.APP_VARIANT === 'production'
+    ? projectName
+    : `${projectName}.${process.env.APP_VARIANT}`;
+
+export default {
+  name: appName,
+  slug: projectName,
+  scheme: appName,
+  version: '1.0.0',
+  orientation: 'landscape',
+  icon: './assets/icon.png',
+  userInterfaceStyle: 'dark',
+  splash: {
+    image: './assets/splash.png',
+    resizeMode: 'contain',
+    backgroundColor: '#ffffff',
+  },
+  assetBundlePatterns: ['**/*'],
+  ios: {
+    supportsTablet: true,
+    requireFullScreen: true,
+  },
+  android: {
+    adaptiveIcon: {
+      foregroundImage: './assets/adaptive-icon.png',
+      backgroundColor: process.env.ICON_COLOR || '#ffffff',
+    },
+    package: `com.craigmulligan.${appName}`,
+    config: {
+      googleMaps: {
+        apiKey: process.env.GOOGLE_MAP_API_KEY,
```

Comment:
did you get the maps to work in your tests? I used a new Gmaps key with `GOOGLE_MAP_API_KEY=deadbeaf yarn run start` but it didn't seem to pick it up

Reply:
The key is loaded in during the native build. Without this key in the config you get an alert saying api key is missing. But the map has never worked for me, maybe because I was using your old revoked key?

Reply:
Yeah it hasn’t worked for me since I revoked the key but tried swapping in a new key and it was the same. Not an issue for now anyhow